server/lib/404.js

```diff
@@ -6,6 +6,7 @@
 
 // It's a 404 not found response.
 
+/*jshint unused: false */
 module.exports = function (req, res, next) {
```

Comment:
Tried that but it doesn't work. The presence of the `next` affects how express routes the errors.